### DISCUSSION TITLE
Halmet hardware problem?

### DISCUSSION THREAD

**Discussion by big-col** [hatlabs/discussions#98] [General]
Remember the ticking/clicking sound an old style HDD would make in operation, very quiet but you could hear it....well my Halmet board is making a similar noise...and seems to be time with the blue LED.

Is this normal?  I don't think this was happening last week. 

Colin

**Comment by mairas** [comment]
I have noticed the same in some conditions. It's probably not related to the LED, but depending on the software you're running, the LED flashing might be in response to packets sent over WiFi. That takes a lot of power, so it might be some feature in the power supply design that is causing the sound. The same ticking noise has been present in some other designs that have been running for years non-stop, so I wouldn't be concerned.

**Comment by mairas** [comment]
Re-opening to provide visibility for future readers.
